saleor/product/utils.py

```diff
@@ -97,32 +99,61 @@ def products_for_cart(user):
     return products
 
 
-def get_variant_picker_data(variants, attributes):
-    data = {'attributes': [], 'variants': []}
-    for attribute in attributes:
-        data['attributes'].append({
+def get_variant_picker_data(product, discounts=None):
+    availability = get_availability(product, discounts)
+    variants = product.variants.all()
+    data = {'productAttributes': [], 'variantAttributes': [], 'variants': []}
```

Comment:
Product attributes don't change though, can't we render them in the template? Do we need the entire sidebar to be a single react component?